gym/lib/gym/detect_values.rb

```diff
@@ -94,6 +98,8 @@ def self.detect_selected_provisioning_profiles
           project = Xcodeproj::Project.open(project_path)
           project.targets.each do |target|
             target.build_configuration_list.build_configurations.each do |build_configuration|
+              next unless build_configuration.to_s == Gym.config[:configuration]
```

Comment:
It indicates that `Gym` skip unused configuration.
If I set `configuration: 'Debug'` in Gym option, other configration (like `Release`) is unnessesary to archive package.